src/instruments/src/EFB/Ground/Pages/Payload/Payload.tsx

```diff
@@ -758,7 +786,14 @@ export const Payload = () => {
                         <div className="flex flex-row mt-4">
                             <Card className="w-full h-full" childrenContainerClassName="flex flex-col w-full h-full">
                                 <div className="flex flex-row justify-between items-center">
-                                    <div className="flex font-medium">Loading Time </div>
+                                <div className="flex font-medium">
++                                   {t('Ground.Payload.BoardingTime')}
++                                    <span className="flex relative flex-row items-center ml-2 text-sm font-light">
++                                        (
++                                        {`${calculateBoardingTime()} ${t('Ground.Payload.EstimatedDurationUnit')}`}
++                                        )
++                                    </span>
++                                </div>
```

Comment:
Somehow the + sign from the diff ended up here
```suggestion
                                    {t('Ground.Payload.BoardingTime')}
                                     <span className="flex relative flex-row items-center ml-2 text-sm font-light">
                                         (
                                         {`${calculateBoardingTime()} ${t('Ground.Payload.EstimatedDurationUnit')}`}
                                         )
                                     </span>
                                 </div>
```